### ISSUE TITLE
Can't work when all layers are unfrozen!!!

### ISSUE THREAD

**Issue by Richardkang1992** [sthanhng/yolov3-tensorflow#1]
Dear sthanh:

There is problem when I used your programming: after that all layers were unfrozen (start from 'iteration : 16'), both 'training loss' and 'validation loss' became '0.0' or 'nan'.  So, how can I make it work in my project availably?

Hopefully that you can give  me some advice!

Wish you bests!

The problem is showed as below:
"
| iteration: 13, training loss: 6721.94, validation loss: 6659.01
#####################################
| iteration: 14, training loss: 6678.08, validation loss: 6636.83
#####################################
| iteration: 15, training loss: 6459.7, validation loss: 6607.77
#####################################
| iteration: 16, training loss: 0.0, validation loss: 0.0
#####################################
| iteration: 17, training loss: 0.0, validation loss: 0.0
#####################################
| iteration: 18, training loss: 0.0, validation loss: 0.0
#####################################
| iteration: 19, training loss: 0.0, validation loss: 0.0
#####################################
| iteration: 20, training loss: 0.0, validation loss: 0.0
#####################################
| iteration: 21, training loss: nan, validation loss: 0.0
#####################################
| iteration: 22, training loss: nan, validation loss: nan
#####################################
| iteration: 23, training loss: nan, validation loss: nan
"
